server/models/User.js

```diff
@@ -7,6 +7,10 @@ const userSchema = new Schema(
             type: String,
             required: true
         },
+        brand_name: {
```

Comment:
Adding to User instead of `business_brand_name` so works with already existing design.